api/namex/resources/name_requests/report_resource.py

```diff
@@ -279,14 +280,16 @@ def _is_ia(legal_type):
 
 
     @staticmethod
-    def _get_instruction_group(legal_type):
+    def _get_instruction_group(legal_type, request_action):
         if ReportResource._is_modernized(legal_type):
             return 'modernized'
         if ReportResource._is_colin(legal_type):
             return 'colin'
         if ReportResource._is_society(legal_type):
             return 'so'
         if ReportResource._is_ia(legal_type):
+            if request_action == RequestAction.NEW.value:
+                return 'colin'
             return 'ia'
```

Comment:
How about a code comment here to explain the logic -- why are you returning "colin" if it's a CR/UL/CC IA?

Reply:
Fixed

Reply:
You fixed the bug but didn't add a comment.

Do you think it's clear enough what is being done here?